x/auth/ante/testutil_test.go

```diff
@@ -113,7 +114,9 @@ func (suite *AnteTestSuite) CreateTestTx(privs []crypto.PrivKey, accNums []uint6
 			AccountNumber:   accNums[i],
 			AccountSequence: accSeqs[i],
 		}
-		sigV2, err := tx.SignWithPrivKey(suite.clientCtx.TxConfig.SignModeHandler().DefaultMode(), signerData, suite.txBuilder, priv, suite.clientCtx.TxConfig)
+		sigV2, err := tx.SignWithPrivKey(
```

Comment:
We don't need `accSeqs[i]` as that data is in `signerData`

Reply:
https://github.com/cosmos/cosmos-sdk/pull/6997#discussion_r470629273